events/wc_offmap_events.txt

```diff
@@ -706,34 +882,614 @@ narrative_event = {
+       text = EVTDESC_WCOME.360_cthun
+       picture = GFX_evt_cthun
+    }
+	desc = {
+       trigger = { ROOT = { has_character_modifier = follower_of_yogg_saron } }
+       text = EVTDESC_WCOME.360_yoggsaron
+       picture = GFX_evt_yogg_saron
+    }
+	desc = {
+       trigger = { ROOT = { has_character_modifier = follower_of_nzoth } }
+       text = EVTDESC_WCOME.360_nzoth
+       picture = GFX_evt_nzoth
+    }
+	border = GFX_event_narrative_frame_war
+
+	is_triggered_only = yes
+
+	major = yes
+	major_trigger = {
+		ai = no
+	}
+	show_root = yes
+
+	immediate = {
+		if = {
+			limit = { has_character_flag = freeing_of_cthun_flag }
+			1613 = { save_event_target_as = target_province }
+		}
+		if = {
+			limit = { has_character_flag = freeing_of_nzoth_flag }
+			2201 = { save_event_target_as = target_province }
+		}
+		if = {
+			limit = { has_character_flag = freeing_of_yogg_saron_flag }
+			858 = { save_event_target_as = target_province }
+		}
+		
+		if = {
+			limit = {
+				character = ROOT
+				cthun_offmap_decisions_enabled_trigger = yes
+			}
+			
+			# Gives gold
+			wealth = 1000
+			
+			# Spawns troops
+			spawn_cthun_invasion_troops_effect = yes
+			spawn_cthun_invasion_troops_effect = yes
+			spawn_cthun_invasion_troops_effect = yes
+			spawn_cthun_invasion_troops_effect = yes
+			if = {		# Gives troops and gold to another Twin Emperor
+				limit = {
+					k_ahnqiraj = {
+						holder_scope = {
+							reawakening_of_cthun_qiraji_event_trigger = yes
+						}
+					}
+				}
+				k_ahnqiraj = {
+					holder_scope = {
+						# Gives gold
+						wealth = 500
+						
+						# Spawns troops
+						spawn_cthun_invasion_troops_effect = yes
+						spawn_cthun_invasion_troops_effect = yes
+					}
+				}
+			}
+			else = {
+				spawn_cthun_invasion_troops_effect = yes
+				spawn_cthun_invasion_troops_effect = yes
+			}
+			clr_character_flag = reawakening_of_cthun_flag 
+		}
+		else_if = {
+			limit = {
+				character = ROOT
+				nzoth_offmap_decisions_enabled_trigger = yes
+			}
+			
+			# Gives gold
+			wealth = 1000
+			# Spawns troops
+			spawn_nzoth_invasion_troops_effect = yes
+			spawn_nzoth_invasion_troops_effect = yes
+			spawn_nzoth_invasion_troops_effect = yes
+			spawn_nzoth_invasion_troops_effect = yes
+			spawn_nzoth_invasion_troops_effect = yes
+			spawn_nzoth_invasion_troops_effect = yes
+			clr_character_flag = reawakening_of_nzoth_flag 
+		}
+		else_if = {
+			limit = {
+				character = ROOT
+				yogg_saron_offmap_decisions_enabled_trigger = yes
+			}
+			
+			# Gives gold
+			wealth = 1000
+			# Spawns troops
+			spawn_yogg_saron_invasion_troops_effect = yes
+			spawn_yogg_saron_invasion_troops_effect = yes
+			spawn_yogg_saron_invasion_troops_effect = yes
+			spawn_yogg_saron_invasion_troops_effect = yes
+			spawn_yogg_saron_invasion_troops_effect = yes
+			spawn_yogg_saron_invasion_troops_effect = yes
+			clr_global_flag = reawakening_of_yogg_saron_flag 
+		}
```

Comment:
What is a point of moving it to `immediate`? It makes the event giant.
Why the second twin emperor doesn't get anything unless the first twin emperor doesn't follow C'Thun?
What if the second twin emperor doesn't follow C'Thun?
What if the first twin emperor doesn't follow any Old God.